{"name": "test_get_user", "status": "broken", "statusDetails": {"message": "AttributeError: 'str' object has no attribute 'id'", "trace": "@pytest.fixture(scope=\"module\")\n    def user_data():\n>       return generate_user_data()\n\n..\\conftest.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    def generate_user_data():\n        user = \"user1\"\n>       user.id = generate_id()\nE       AttributeError: 'str' object has no attribute 'id'\n\n..\\config\\generator.py:34: AttributeError"}, "start": 1719729919446, "stop": 1719729919446, "uuid": "676c2707-d284-4163-8dd2-0b133d4c02e8", "historyId": "001f68e0f9c97eab6db29fb6fd8f801b", "testCaseId": "001f68e0f9c97eab6db29fb6fd8f801b", "fullName": "tests.test_user_operations.TestUserOperations#test_get_user", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUserOperations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "7900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}